backend/src/models/user.ts

```diff
@@ -309,19 +375,21 @@ userSchema.methods.sendBetaApprovalEmail = async function (token) {
 };
 
 // Send beta denial email
-userSchema.methods.sendBetaDenialEmail = async function () {
+userSchema.methods.sendBetaDenialEmail = async function (): Promise<void> {
   // Recipient address (user's email)
+  if (this.betaAccessTokenExpires === undefined) { // TODO: probably want to change default rather than have this case
```

Comment:
It seems like the default is `undefined`. So this may just be redundant. May just be able to do the following,

```suggestion
  if (!this.betaAccessTokenExpires) {
```

Reply:
So `(!this.betaAccessTokenExpires)` will evaluate to `true` if `this.betaAccessTokenExpires` is a different falsy value, such as `null` or `0`

`this.betaAccessTokenExpires === undefined` will evaluate to true only if `this.betaAccessTokenExpires = undefined`. If the behavior is that `this.betaAccessTokenExpires` should be set to a date whenever it's a falsy value, then let's go with your change.

Reply:
Now that we have models/user tests, once those are merged with this branch I'll test it out and see if this change would break any of the tests.